apps/desktop/src/lib/components/ProjectSetupTarget.svelte

```diff
@@ -76,6 +76,16 @@
 		if (!selectedBranch) return;
 		dispatch('branchSelected', [selectedBranch.name, selectedRemote.name]);
 	}
+
+	const projectService = getContext(ProjectService);
+	async function deleteProject() {
+		try {
+			await projectService.deleteProject(project.id);
+			await projectService.reload();
+		} catch (err: any) {
+			console.error(err);
+		}
+	}
```

Comment:
Could we perhaps drop the try/catch here, and rely on the uncaught exception handler to display a toast and log the error? Since it's only two lines I would also prefer if we inline click handler.